automator/src/com/testsigma/automator/actions/web/verify/VerifyElementAbsenceAction.java

```diff
@@ -25,7 +25,7 @@ public void execute() throws Exception {
         elementFound = true;
       }
 
-    } catch (AutomatorException | NotFoundException | TimeoutException | InvalidElementStateException e) {
+    } catch (Exception e) {
       log.debug("Expected exception(Verify element not present):", e);
```

Comment:
@manikanta-reddy-t Here you are setting a success message for these specific exception types which is expected. Now after this change, any RuntimeException will also be handled in the same way. I would recommend you to handle the specific exceptions separately